src/fparser/statements.py

```diff
@@ -967,23 +1105,24 @@ def process_item(self):
         self.nature = nature
         i = line.find(',')
         self.isonly = False
-        if i==-1:
+        if i == -1:
             self.name = line
             self.items = []
         else:
             self.name = line[:i].rstrip()
             line = line[i+1:].lstrip()
-            if line.lower().startswith('only') and line[4:].lstrip().startswith(':'):
+            if line.lower().startswith('only') and \
+               line[4:].lstrip().startswith(':'):
                 self.isonly = True
                 line = line[4:].lstrip()[1:].lstrip()
             self.items = split_comma(line, self.item)
         return
 
-    def tofortran(self, isfix=None):
+    def tofortran(self, isfix=None):  # pylint: disable=invalid-name
         tab = self.get_indent_tab(isfix=isfix)
         s = 'USE'
         if self.nature:
-            s += ' ' + self.nature + ' ::'
+            s += ', ' + self.nature + ' ::'
```

Comment:
Found the bug fix! :-) As you've modified this routine please could you add a docstring for it.

Reply:
Thank you, I thought that adding a docstring for the top "class Use(Statement):" would be enough, but now I added them for each method in the class.